db/migrate/20161128201646_create_quote_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateQuoteLikes < ActiveRecord::Migration
+  def change
+    create_table :quote_likes do |t|
+      t.references :user, index: true, foreign_key: true, null: false
+      t.references :quote, index: true, foreign_key: true, null: false
+
+      t.timestamps null: false
+    end
```

Comment:
Yep, that should work. The old model was called `Quote` as well, for filtering the polymorphic type.